install/kots/manifests/gitpod-http-proxy-settings.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   labels:
     app: gitpod
     component: gitpod-installer
+  annotations:
+    kots.io/when: '{{repl ConfigOptionEquals "enable_proxy_settings" "1" }}'
```

Comment:
Could you point me to docs that explain what this annotation does?

Reply:
This includes a resource in the KOTS release, in this case if the `enabled_proxy_settings` config value equals `1` - it'll either resolve `true` or `<blank`> and the annotation includes it if it's `true`

[Docs](https://docs.replicated.com/vendor/packaging-include-resources#include-a-resource)